client/my-sites/sharing/connections/service.jsx

```diff
@@ -47,6 +47,15 @@ import requestExternalAccess from 'lib/sharing';
 import MailchimpSettings, { renderMailchimpLogo } from './mailchimp-settings';
 import config from 'config';
 
+/**
+ * Check if the connection is broken or requires reauth.
+ *
+ * @param {object} connection Publicize connection.
+ * @returns {boolean} True if connection is broken or requires reauthentication.
+ */
+const isConnectionInvalidOrReauth = connection =>
```

Comment:
Nit: `isConnectionInvalidOrReauth` name is ambiguous - `invalid` is an adjective, but I'm not sure which adjective does `reauth` correspond to. Can we make this clearer?

Reply:
Updated